tests/io/test.xpi.js

```diff
@@ -542,19 +532,17 @@ describe('Xpi.getFileAsStream()', function() {
         expect(err.message).toContain('¡hola!');
       });
   });
-
 });
 
-describe('Xpi.getFilesByExt()', function() {
-
+describe('Xpi.getFilesByExt()', function getFilesByExtCallback() {
```

Comment:
FWIW, you could just create a nested ./tests/.eslintrc which disables that one unnamed function rule, and all the other rules from the root .eslintrc file would still be inherited.